src/__integration_tests__/response-body/invoice.response.examples.ts

```diff
@@ -4,7 +4,7 @@ export const multipleInvoices: InvoicesResponse = {
 	Id: '431b0b6a-81bf-4286-82f3-73e6f8c78fd3',
 	Status: 'OK',
 	ProviderName: 'Xero API Previewer',
-	DateTimeUTC: '\/Date(1518298927172)\/',
+	DateTimeUTC: '/Date(1518298927172)/',
```

Comment:
Are you sure about this? The API really does return a string with an escaped backslash. as described in https://developer.xero.com/documentation/api/requests-and-responses.

Reply:
That's actually are automatic fix by ESLint, but yes, I'm pretty sure backslash doesn't need to be escaped here to be threatened as backslash.